docs/develop-logseq.md

```diff
@@ -37,14 +37,15 @@ yarn release
 1. Install npm packages for building the desktop app
 
 ``` bash
-yarn install && cd static && yarn install && cd ..
+yarn install
```

Comment:
```suggestion
yarn install

# If you are updating from an old branch, checkout the newest dependencies as well.
cd static && yarn install && cd ..
```

Reply:
`static/` is made by gulp after doing a `yarn watch`. Would second @andelf's suggetion as we need to leave this in with a comment

Reply:
Indeed. Moving the two lines after `yarn watch` might be better.

Reply:
@andelf @logseq-cldwalker Got it. Thanks for the better update instructions here.